test/integration/acceptance/create_domain_test.cpp

```diff
@@ -13,6 +13,9 @@ using namespace common_constants;
 
 class CreateDomain : public AcceptanceFixture {
  public:
+  static constexpr iroha::StorageType storage_types[] = {
```

Comment:
I'd better use additional enum value kMAX. instead of enumerating every.

Reply:
Why better? It seems code will become much worst with kMax.